app/models/bot.rb

```diff
@@ -106,4 +119,32 @@ def right_answer(message)
       new_response
     end
   end
+
+  def default_vocabulary(vocabularies)
+    if !vocabularies.nil?
+      vocabularies << "default"
+      vocabularies.each do |vocabulary|
+        csv_text = File.read(Rails.root.join('app', 'assets', "vocabularies", (vocabulary += ".csv")))
```

Comment:
Useless assignment to variable - `vocabulary`.<br>Line is too long. [102/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.